build.sbt

```diff
@@ -32,23 +32,23 @@ ThisBuild / githubWorkflowBuild := Seq(
   )
 )
 
+ThisBuild / githubWorkflowJavaVersions := List(JavaSpec.temurin("8"))
```

Comment:
Thinking about this ... I believe we should probably just use `11` but still target `8`